ibis/expr/api.py

```diff
@@ -51,6 +54,15 @@
     from ibis.expr.schema import SchemaLike
 
 __all__ = (
```

Comment:
I ran `:sort` on this list in vim, which is why there's a bunch of extra movement here.

Turns out we were already exporting `Expr` and `Schema` types (but no others), but these were buried in the `__all__` list before :).